src/Combinatorics/Graphs/functions.jl

```diff
@@ -35,6 +35,68 @@ function Graph{T}(nverts::Int64) where {T <: Union{Directed, Undirected}}
     return Graph{T}(pmg)
 end
 
+@doc raw"""
+    graph_from_adjacency_matrix(::Type{T}, G) where {T <:Union{Directed, Undirected}}
+
+Return the graph with adjacency matrix `G`.
+
+This means that the nodes ``i > j`` are connected by an edge
+if and only if ``G_{i,j}`` is one.
+
+# Examples
+```jldoctest
+julia> G = ZZ[0 0; 1 0]
+[0   0]
+[1   0]
+
+julia> graph_from_adjacency_matrix(Directed, G)
+Graph{Directed}(pm::graph::Graph<pm::graph::Directed>
+{}
+{0}
+)
+
+julia> graph_from_adjacency_matrix(Undirected, G)
+Graph{Undirected}(pm::graph::Graph<pm::graph::Undirected>
+{1}
+{0}
+)
+
+```
+"""
+graph_from_adjacency_matrix(::Type, G::Union{MatElem, Matrix})
+
```

Comment:
Is this line a leftover or does it have some important meaning that I have missed?

Reply:
It serves as an anchor for the documentation. It allows me to document both methods at the same time.